### PR TITLE
Update release_notes.md

### PR DESCRIPTION
**Pull request by trusli**
Add 3.3 release notes

### CODE REVIEW

**Comment by sashadt** [comment]
Going to re-work it?

File: docs/release_notes.md [hunk 1]
```diff
@@ -1,6 +1,227 @@
...
+
+#### dtDasbhoard
+* New widgets for the dashboard: Geo coordinates with circles, Geo regions with gradient fill and single Value 
+
+#### dtAssemble (beta)
+* Top level “Develop” takes users directly to Application Page
+* Navigation changes on how user get to Tuple Schema. User goes to It is now "Edit Application" then "Tuple Schema"
+* Development (breadcrumb link)
+
+#### Document section
```
Comment (sashadt) [inline docs/release_notes.md]: Should this be "Documentation"?

File: docs/release_notes.md [hunk 2]
```diff
@@ -1,6 +1,227 @@
...
+#### Apache Apex 3.3 
+* Support for iterative processing. It is a building block to support machine learning
+* Ability to populate DAG at application launch time
+* Pre checkpoint operator callback so that it can execute a logic before the operator gets checkpointed (e.g. flush file to HDFS)
+* Provide the option for operator to do checkpointing in a distribute in-memory store. It is faster than HDFS due to disk i/o latency
+* Add group ID information in an applicatin package.  It is visible for application grouping in dtHub.
+
+### Appendix
+
+#### Known Issues
```
Comment (sashadt) [inline docs/release_notes.md]: Is this a list of issues not addressed by 3.3 release?

File: docs/release_notes.md [hunk 3]
```diff
@@ -1,6 +1,227 @@
...
+* [SPOI-7604] - HDHT bucket meta class is obfuscated incorrectly
+* [SPOI-7615] - dtHub intermittently throws error as "Failed to import" while importing multiple pkgs at the same time 
+* [SPOI-7619] - specify bower link malhar-angular-dashboard to version 1.0.1
+* [SPOI-7644] - Need to update installer script
+* [SPOI-7679] - Unable to access newly added features for Community edition.
+
+### Apache Apex 3.3 Change Logs
+https://github.com/apache/incubator-apex-core/blob/v3.3.0-incubating/CHANGELOG.md
+
+### New Feature
```
Comment (sashadt) [inline docs/release_notes.md]: Should this, and few following headings be a sub-section under Apache Apex 3.3 Change Logs?

File: docs/release_notes.md (hunk 1 above)
Comment (trusli) [inline docs/release_notes.md]: yes

File: docs/release_notes.md (hunk 2 above)
Comment (trusli) [inline docs/release_notes.md]: yes

File: docs/release_notes.md (hunk 3 above)
Comment (trusli) [inline docs/release_notes.md]: yes. that's better

**Pull request merged**